### PR TITLE
reflect changes from repositories

### PR DESCRIPTION
**Pull request by j33f**

### CODE REVIEW

File: source/includes/_securityController.md [hunk 1]
```diff
@@ -1327,9 +1312,7 @@ Retrieves a list of `role` objects from Kuzzle's database layer given a list of
 ```litcoffee
 {
   body: {
-    // if hydrate is true, the object is returned as Profile object (with an array of Role object) instead of plain elasticsearch document
-    hydrate: false,
-    // filter can contains an array `roles` with a list of role id
+   // filter can contains an array `roles` with a list of role id
```
Comment (scottinet) [inline source/includes/_securityController.md]: "A `roles` can be added containing a list of role IDs"

File: source/includes/_securityController.md [hunk 2]
```diff
@@ -1351,9 +1334,7 @@ Retrieves a list of `role` objects from Kuzzle's database layer given a list of
   controller: "security",
   action: "searchProfiles",
   body: {
-    // if hydrate is true, the object is returned as Profile object (with real Role object) instead of plain elasticsearch document
-    hydrate: false,
-    // filter can contains an array `roles` with a list of role id
+    // fiter can contains an array `roles` with a list of role id
```
Comment (scottinet) [inline source/includes/_securityController.md]: Proposal: "A `roles` array can be added containing a list of role IDs"
If you prefer your original sentence, then you have to remove the 's' at the end of "contains"

File: source/includes/_securityController.md [hunk 3]
```diff
@@ -1368,13 +1349,11 @@ Retrieves a list of `role` objects from Kuzzle's database layer given a list of
   controller: "security",
   action: "searchProfiles",
   body: {
-    // if hydrate is true, the object is returned as Profile object (with real Role object) instead of plain elasticsearch document
-    hydrate: false,
     // filter can contains an array `roles` with a list of role id
```
Comment (scottinet) [inline source/includes/_securityController.md]: Proposal: "A `roles` array can be added containing a list of role IDs"
If you prefer your original sentence, then you have to remove the 's' at the end of "contains"

File: source/includes/_securityController.md [hunk 4]
```diff
@@ -1351,9 +1334,7 @@ Retrieves a list of `role` objects from Kuzzle's database layer given a list of
   controller: "security",
   action: "searchProfiles",
   body: {
-    // if hydrate is true, the object is returned as Profile object (with real Role object) instead of plain elasticsearch document
-    hydrate: false,
-    // filter can contains an array `roles` with a list of role id
+    // fiter can contains an array `roles` with a list of role id
    roles:  ['myrole', 'admin'],
    // filter can handler pagination with properties `from` and `size`
```
Comment (scottinet) [inline source/includes/_securityController.md]: "filter can handle pagination using the `from` and `size` properties"

File: source/includes/_securityController.md [hunk 5]
```diff
@@ -1390,13 +1369,11 @@ Retrieves a list of `role` objects from Kuzzle's database layer given a list of
...
   body: {
-    // if hydrate is true, the object is returned as Profile object (with real Role object) instead of plain elasticsearch document
-    hydrate: false,
     // filter can contains an array `roles` with a list of role id
-   roles:  ['myrole', 'admin'],
-   // filter can handler pagination with properties `from` and `size`
-   from: 0,
-   size: 10
+    roles:  ['myrole', 'admin'],
+    // filter can handler pagination with properties `from` and `size`
```
Comment (scottinet) [inline source/includes/_securityController.md]: "filter can handle pagination using the `from` and `size` properties"

File: source/includes/_securityController.md [hunk 6]
```diff
@@ -1390,13 +1369,11 @@ Retrieves a list of `role` objects from Kuzzle's database layer given a list of
   controller: "security",
   action: "searchProfiles",
   body: {
-    // if hydrate is true, the object is returned as Profile object (with real Role object) instead of plain elasticsearch document
-    hydrate: false,
     // filter can contains an array `roles` with a list of role id
```
Comment (scottinet) [inline source/includes/_securityController.md]: Proposal: "A `roles` array can be added containing a list of role IDs"
If you prefer your original sentence, then you have to remove the 's' at the end of "contains"

File: source/includes/_securityController.md [hunk 7]
```diff
@@ -1368,13 +1349,11 @@ Retrieves a list of `role` objects from Kuzzle's database layer given a list of
...
-   roles:  ['myrole', 'admin'],
-   // filter can handler pagination with properties `from` and `size`
-   from: 0,
-   size: 10
+    roles:  ['myrole', 'admin'],
+    // filter can handler pagination with properties `from` and `size`
```
Comment (scottinet) [inline source/includes/_securityController.md]: "filter can handle pagination using the `from` and `size` properties"

File: source/includes/_securityController.md [hunk 8]
```diff
@@ -1446,10 +1423,8 @@ Retrieves a list of `role` objects from Kuzzle's database layer given a list of
 }
 ```
 
-Retrieves all the profiles that contain a given set of roles.  
+Retrieves all the profiles that contain a given set of roles into their policies.
```
Comment (scottinet) [inline source/includes/_securityController.md]: "Retrieves profiles referring to a given set of roles in their policies"

**Comment by stafyniaksacha** [comment]
+1

**Pull request merged**